### ISSUE TITLE
Unity Fixture output format

### ISSUE THREAD

**Issue by rollebolle** [ThrowTheSwitch/Unity#271]
The output format of unity fixtures is different from Unity without fixtures. This creates problems with the python scripts like unity_to_junit.py. It would be nice to either have the option to conform to normal Unity format, or if extra scripts was added for this translation.

I have a crude script unity_fixtures_to_junit.py. for this that I could PR if you think it is a good idea.

**Comment by mvandervoord** [comment]
Yeah, I like the idea of having a separate script for reading fixtures output and putting it into the same output. This allows the fixtures users to continue to have their output match cppuTest, which is the primary purpose of the fixtures addon.

**Comment by mvandervoord** [comment]
Seems like we should implement this option along with #270 together.

**Comment by ssj71** [comment]
@rollebolle is that script anywhere public? I need something like that, even if its crude. :)
